packages/web-client/tests/unit/helpers/share/role.spec.ts

```diff
@@ -16,266 +16,268 @@ import {
   SharePermissions
 } from '../../../../src/helpers/share'
 
-describe('ShareRole', () => {
-  describe('permissions', () => {
-    it('returns full permissions of a role that has no share permission', () => {
-      const permissions = [SharePermissions.read, SharePermissions.update]
-      const role = new PeopleShareRole('', true, '', '', '', permissions)
-      expect(role.permissions(false)).toEqual(permissions)
-      expect(role.permissions(true)).toEqual(permissions)
-    })
-    it.each([
-      [
-        'allowed',
-        {
-          permissions: [SharePermissions.read, SharePermissions.update, SharePermissions.share],
-          allowSharing: true,
-          result: [SharePermissions.read, SharePermissions.update, SharePermissions.share]
-        }
-      ],
-      [
-        'not allowed',
-        {
-          permissions: [SharePermissions.read, SharePermissions.update, SharePermissions.share],
-          allowSharing: false,
-          result: [SharePermissions.read, SharePermissions.update]
-        }
-      ]
-    ])(
-      'respects that re-sharing is %s for a role that has share permissions',
-      (name, { permissions, allowSharing, result }) => {
+describe('roles', () => {
```

Comment:
The file only has indentation changes because I added this wrapping `describe` block. Before it was not possible to run all unit tests in this file at once from within a jetbrains IDE.